core/src/main/resources/org/apache/spark/ui/static/stagepage.js

```diff
@@ -1079,15 +1069,8 @@ $(document).ready(function () {
             }
           ],
           "columnDefs": [
-            { "visible": false, "targets": 11 },
-            { "visible": false, "targets": 12 },
-            { "visible": false, "targets": 13 },
-            { "visible": false, "targets": 14 },
-            { "visible": false, "targets": 15 },
-            { "visible": false, "targets": 16 },
-            { "visible": false, "targets": 17 },
-            { "visible": false, "targets": 18 },
-            { "visible": false, "targets": 21 }
+            { "visible": false, "targets": optionalColumns },
+            { "visible": false, "targets": 18 }, // accumulators
```

Comment:
does it affect the display ordering, before is .`.., 18, 21`, now is `..., 21, 18`

Reply:
no